src/Compiler/Interactive/fsi.fs

```diff
@@ -1410,6 +1411,268 @@ type internal FsiConsolePrompt(fsiOptions: FsiCommandLineOptions, fsiConsoleOutp
 
     member _.FsiOptions = fsiOptions
 
+//----------------------------------------------------------------------------
+// fsi #h
+//----------------------------------------------------------------------------
+module Fsih =
```

Comment:
Probably should be more verbose, something like `FsHelp` or something along the lines.